subcommands/waves/init.go

```diff
@@ -28,9 +29,15 @@ This command only initializes a wave, but does not provision its updates to devi
 Use a "fioctl wave rollout <wave> <group>" to trigger updates of this wave to a device group.
 Use a "fioctl wave complete <wave>" to update all devices (make it globally available).
 Use a "fioctl wave cancel <wave> to cancel a wave (make it no longer available).`,
+		PreRun: func(cmd *cobra.Command, args []string) {
+			confirmWaveInitProceed()
+		},
 		Run:  doInitWave,
 		Args: cobra.ExactArgs(3),
 		Example: `
+**(Recommended)** Generate OSTree static deltas before rolling out waves to devices.
+More info: https://docs.foundries.io/latest/reference-manual/ota/static-deltas.html#ref-static-deltas
+
```

Comment:
I don't think this is an appropriate place to put this.
Maybe, it might fit at the end of the `Long` help...

But we decided to print it at the end of command execution; not sure we need to duplicate it in two places.
That might scare the user from using Waves instead of encouraging them to use Static Deltas.

@doanac @mike-sul your thoughts.